bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestCheckIssueVisitor.java

```diff
@@ -162,12 +162,16 @@ public void visit(CommitterEmailIssue issue) {
 
     @Override
     public void visit(AuthorNameIssue issue) {
-        throw new IllegalStateException("Invalid author name: " + issue.commit().author());
+        // We only get here for contributors without an OpenJDK username
+        addFailureMessage(issue.check(), "Pull request's HEAD commit must contain a full name");
```

Comment:
This is not strictly true, CheckRun.java around line 194 will check that all commits in a PR contain the same name and email address. Could of course start using just the one in HEAD without that check..

Reply:
Thanks, updated in b49bd90